src/openforms/logging/logevent.py

```diff
@@ -441,18 +449,94 @@ def submission_export_list(form: "Form", user: "User"):
+    keys_mapped_to_component = {}
+    for formstep in formsteps:
+        keys_mapped_to_form = {
+            **{component["key"]: formstep for component in formstep.iter_components()},
+            **keys_mapped_to_form,
+        }
+        keys_mapped_to_component = {
+            **flatten(formstep.form_definition.configuration),
+            **keys_mapped_to_component,
+        }
 
+    involved_components = {}
     for evaluated_rule in evaluated_rules:
 
         rule = evaluated_rule["rule"]
+        logic_data = JsonLogicTest.from_expression(rule.json_logic_trigger)
+        source_components = convert_json_logic_in_json_tree(
+            logic_data,
+            keys_mapped_to_component,
+            keys_mapped_to_form,
+            updated_submission_data,
+        )
+
+        stringified_json_tree = stringify_json_tree(source_components)
+
+        # will remove duplicate components that are involved in several rules
+        remove_duplicate_components_from_json_tree(
+            source_components, involved_components
+        )
+        targeted_components = []
+        for action in rule.actions:
+            log_action = {}
+
+            # Action verifications
+            action_details = action["action"]
+            log_action["value"] = ""
+            log_action["type"] = action_details["type"]
+            if action_details["type"] == LogicActionTypes.value:
+                log_action["value"] = action_details["value"]
+                log_action["key"] = action["component"]
+            elif action_details["type"] == LogicActionTypes.property:
+                log_action["value"] = action_details["property"]["value"]
+                log_action["state"] = action_details["state"]
+                log_action["key"] = action["component"]
+            elif action_details["type"] == LogicActionTypes.variable:
+                log_action["key"] = action["variable"]
+                logic = JsonLogicTest.from_expression(action_details["value"])
+                src_components = convert_json_logic_in_json_tree(
+                    logic,
+                    keys_mapped_to_component,
+                    keys_mapped_to_form,
+                    updated_submission_data,
+                )
+                log_action["value"] = stringify_json_tree(src_components)
+                log_action["isVariable"] = True
+            targeted_components.append(log_action)
+
+        # Assuming key is unique in all form steps
+        for targeted_component in targeted_components:
+            key = targeted_component.get("key")
+            # Adding the label of the key and step name associated if key exists
+            if not key:
+                continue
+            # Might happens that a form variable is not related to a component
+            if targeted_component.get(
+                "isVariable"
+            ) and not keys_mapped_to_component.get(key):
+                targeted_component["label"] = key
```

Comment:
we should look up the form variables in that case and grab the name of the variable